modules/tile-converter/src/lib/utils/geometry-utils.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * luma.gl can not work without indices now:
+ * https://github.com/visgl/luma.gl/blob/d8cad75b9f8ca3e578cf078ed9d19e619c2ddbc9/modules/experimental/src/gltf/gltf-instantiator.js#L115
```

Comment:
This seems to be a serious bug in luma.gl and should be fixed there. Geometries without indices are quite common, in fact for advanced 3D it is usually not desirable to share vertices as normals tangents etc can vary even if the position is the same.